2019/April/community_news.md

```diff
@@ -26,6 +26,6 @@
   Chennai Trade Center and IIT Madras Research Park  
   https://in.pycon.org/2019/
 
-* DGPLUG Summer Training 2019
+* DGPLUG Summer Training-2019
```

Comment:
For reviewer:
This change just to include file :)

Reply:
Can you please revert the changes as we can see from this job it looks good! 
https://github.com/pythonpune/meetup-talks/pull/71/checks?sha=08b640a3c6743f1ea16754f8922bc29a299483f5